Yoast/wordpress-seo
# Remove configuration wizard package

## Context


* Cleanup repo: we moved away from this package, I think when we made the first time configuration

## Summary


This PR can be summarized in the following changelog entry:

* Removes `@yoast/configuration-wizard` from the repository.

## Relevant technical choices:

* Folder was added to https://github.com/Yoast/archive in https://github.com/Yoast/archive/commit/13db04df6d2661337d4034e860dbdd7ad199ea3f
* I left the `yoast-components` CSS file, even though it states at the top it is only for the configuration wizard. This makes more sense to me to remove within the scope of the `yoast-components` package.
* I left the `apps/components` because that app does not run anyway, so I have no way of testing.  We should probably remove it too 😉 
* Holding of on the NPM deprecation due to the dependency in `yoast-components that is removed in this PR. I added it to the issue of the removal of `yoast-components` instead

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Check if you can still build
* Quick sanity check along some pages with the console open

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Quick sanity check along some pages with the console open

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* Nothing should be impacted, since we don't use it

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes https://github.com/Yoast/wordpress-seo/issues/19918
